apps/condo/domains/billing/utils/testSchema/index.js

```diff
@@ -378,6 +418,31 @@ async function makeClientWithIntegrationAccess () {
     return client
 }
 
+/**
+ * Simplifies creating series of instances
+ */
+
+async function makeContextWithOrganizationAndIntegrationAsAdmin() {
```

Comment:
usually, we use `make<anythong>Client` for client factory logic. Probably it's just a `create` helper.